lib/elixir/test/erlang/tokenizer_test.erl

```diff
@@ -5,6 +5,10 @@ tokenize(String) ->
   {ok, _Line, _Column, Result} = elixir_tokenizer:tokenize(String, 1, []),
   Result.
```

Comment:
Let's change `tokenize/1` so that its implementation becomes just a call to `tokenize/2`:

```erlang
tokenize(String) ->
  tokenize(String, []).
```